docroot/themes/humsci/su_humsci_subtheme/Gruntfile.js

```diff
@@ -0,0 +1 @@
+../su_humsci_theme/Gruntfile.js
```

Comment:
Is this subtheme just for testing?

Reply:
Its only used for stubbing out new subthemes. should i add `hidden: true` to the info on this subtheme?